preprocessors/openstreetmap/main.py

```diff
@@ -160,5 +166,52 @@ def get_map_data():
     return response
 
 
+def unmodified_street_data(OSM_data):
+    """
+     The arguement of this function is the raw street data from
+     python overpy API, but the data cannot be accessed directly as
+     it only represents the memory address of the actual result,
+     which is also a list object. What this function does however is
+     to slightly modify it in a similar manner as would be
+     obtained directly from OVERPASS API.
+     For more information see-
+     https://python-overpy.readthedocs.io/en/latest/api.html.
+
+    """
+    elements = []
+    nodes_list = []
+    # List all ways with their respective nodes as
+    # in OVERPASS API
+    for way in OSM_data.ways:
```

Comment:
Is this just manual serialization? If so, is there a way to load this data _back into_ this library so we can traverse data that way? If not, there really isn't a point to including this new field.

Reply:
Yes. I did a lot of research on that, but could not find any workable approach to it. Unfortunately, the documentation for the api only speaks to iterating over the data to extract any items of interest (which is what I simply did here). Alternatively, I'm thinking maybe I would have to use the Python's Pickle library directly on the API output, then provide short guidelines on how to deserialise it at the handler's side. But I don't know if that is going to work yet except someone tries there after I've done it.

Reply:
Please figure out if there's a feasible approach, whether it's this or another one. If there's no way to export and retrieve the data (which _would_ come as a surprise to me), then we need to change our approach.